azure-devops/azext_devops/dev/team/_help.py

```diff
@@ -28,6 +28,16 @@ def load_team_help():
     short-summary: Manage service endpoints/service connections
     """
 
+    helps['devops service-endpoint azurerm'] = """
+    type: group
+    short-summary: Manage Azure RM service endpoints/service connections
+    """
```

Comment:
nit: service endpoints/connections 
For all help texts.. confirm with @geverghe.

Reply:
@atbagga - thanks for the catch. And yes, since we are breaking away into new commands can we stick with the product nomenclature rather than the API nomenclature? I do not have any data points/feedback for this, but I personally feel better about sticking with the product nomenclature since we remain consistent. @gauravsaralMs - let me know your thoughts.

Reply:
If at all we have to point users to read response/request docs then they will read service endpoint 
also this change will flow across to other commands/code which will be a significant change.. 
Let me know what will be better for customers and we can do that

Reply:
Ohh.. My original comment was only meant for a minor text change.  **service endpoint/ service connection**  should be **service endpoint/connection.** :)

I agree with Gaurav on not trying to change the name since the API docs will always call it service-endpoint anyways. Just because UI label was renamed we will end up maintaining a deprecation cycle for a set of commands doesn't feel right.

Reply:
Pretty late to this party - but my reasoning as below - @atbagga , @gauravsaralMs :
1. Reference the User Experience - UI, when referencing to labeling as fundamentally terminology is the CLI experience and therefore it has to be consistent across interaction mediums. In case of conflicts with REST API, I think it would be better to adhere to the UI over the REST API as UI is the predominant interaction medium. 
2. My initial thought was with the breaking of the command, the usage pattern has changed; therefore, you are breaking the commanding pattern at this point, then include the name change as well. The deprecation pattern of hiding the command in the current version preventing new intake but supporting the existing users ensures that we are compliant. In terms of maintenance, if the existing users require any additional features, experience, or fixes, we point them to the new command. Feel that this is an acceptable experience.

Since this release has gone, I guess it is a wait till some conflict or feedback happens.